update_properties.rb

```diff
@@ -0,0 +1,44 @@
+##
+## This utility makes it easy to convert the properties from the default properties file to use confd properties
```

Comment:
Ok, I'll remove.  

I've placed it here instead: https://gist.github.com/temyers/795f3f89ac94e8be85b4f9e1c6f298c0